compiler/test/src/test/java/org/apache/nemo/compiler/frontend/beam/transform/GroupByKeyAndWindowDoFnTransformTest.java

```diff
@@ -88,13 +89,15 @@ public void test() {
     final SlidingWindows slidingWindows = SlidingWindows.of(Duration.standardSeconds(2))
       .every(Duration.standardSeconds(1));
 
-    final GroupByKeyAndWindowDoFnTransform<String, String> doFnTransform =
-      new GroupByKeyAndWindowDoFnTransform(
+    final GBKTransform<String, String, Iterable<String>> doFnTransform =
```

Comment:
Do we need `GroupByKeyAndWindowDoFnTransformTest`? Can we merge this test with `GBKTransformTest`?

Reply:
No, I'll merge!